schemas/dex/schema.json

```diff
@@ -246,6 +246,10 @@
           "description": "The amount of the output token, decimal normalized.",
           "type": "number"
         },
+        "spot_price_after_swap": {
+          "description": "The spot price in the pool after the swap is complete.",
```

Comment:
Should this be in USD? Or is there another denomination across all dexs?

Reply:
Let's keep it not optional, if a protocol doesn't have this available, they can mark it null or 0

Reply:
It's a ratio between the two tokens. v3 DEXs show the tick after the swap, which tells us the price ratio.

Reply:
Can we add something about this to the description?